File: safari-600.7-branch/LayoutTests/http/tests/security/resources/no-javascript-location.php
<?php
  header('HTTP/1.0 200 OK');
  header('Content-type: text/html');
  header('Location: javascript:window.top.location="about:blank"');
?>
<!DOCTYPE html>
<body>
<script>
if (window.testRunner)
    testRunner.dumpAsText();
</script>
FAIL: This content should not appear after a failed Location redirect to a javascript: URL.
</body>

File: LayoutTests/http/tests/security/resources/no-javascript-location.py
#!/usr/bin/env python3

import sys

sys.stdout.write(
    'status: 200\r\n'
    'Location: javascript:window.top.location="about:blank"\r\n'
    'Content-Type: text/html\r\n\r\n'
    '<!DOCTYPE html>\n'
    '<body>\n'
    '<script>\n'
    'if (window.testRunner)\n'
    '    testRunner.dumpAsText();\n'
    '</script>\n'
    'FAIL: This content should not appear after a failed Location redirect to a javascript: URL.\n'
    '</body>\n'
)